ui-components/src/global/forms.scss

```diff
@@ -321,7 +321,6 @@ input[type="number"] {
 .field-note {
   @apply text-tiny;
   @apply text-gray-700;
-  @apply font-semibold;
```

Comment:
Is this font weight change and the one for AlertBox something that's needed specifically for SF? Do we know if we can update the Core design?

Reply:
Yep! This creates inconsistency in core